frontend/src/components/LoadingButton.tsx

```diff
@@ -0,0 +1,21 @@
+// put in a component instead of its own asset file, because we need tailwind classes
+const ButtonSpinner = () => (
```

Comment:
nice refactor! think we can just call this guy `Button` and use it throughout our app (for withdraw, other modals, with custom styling)

Reply:
renamed